packages/terra-overlay/src/LoadingOverlay.jsx

```diff
@@ -12,25 +12,29 @@ const { BackgroundStyles, zIndexes } = Overlay.Opts;
 
 const propTypes = {
   /**
- * The message to be displayed within the overlay.
- */
-  message: PropTypes.string,
+   * The visual theme to be applied to the overlay background. Accepts 'light', 'dark', and 'clear' or BackgroundStyles.LIGHT, BackgroundStyles.DARK, and BackgroundStyles.CLEAR.
+   */
+  backgroundStyle: PropTypes.oneOf(['light', 'dark', 'clear', BackgroundStyles]),
   /**
-  * Indicates if the icon spinner should be animated.
-  */
-  isAnimated: PropTypes.bool,
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```

Comment:
We should add the `@private` syntax so this doesn't show up in the generated prop-types documentation.